src/TaylorInversion.jl

```diff
@@ -101,7 +93,6 @@ end
 
 function further_substitution(it::InverseTaylor{N}, subbed::Num) where {N}
     for i in 1:N
-        @info "Expanding term $i to create ivnersion expression"
```

Comment:
Do I understand correctly that these `@info`s are removed for the sake of keeping the `@generated` function pure?

Reply:
Yes, printing is not allowed from inside generated functions.